package.json

```diff
@@ -17,6 +17,15 @@
     "test262-harness": "^6.2.0"
   },
   "scripts": {
-    "test": "./tools/scripts/ci_test.sh"
+    "ci": "./tools/scripts/ci_test.sh",
+    "jsvu": "jsvu",
```

Comment:
This seems unnecessary

Reply:
I only had it in a hope to eventually being able to hook jsvu usage in a local folder, so I could call a npm script to install engines locally as dependencies before running the tests.

So far, jsvu serves the CircleCI project, otherwise it's hard to convince anyone for other usages.

Reply:
I’m not familiar with your particular setup but it really sounds like it should be possible to access and run things in your home directory, even within [the CircleCI environment](https://circleci.com/docs/2.0/circleci-images/). I’m hoping we can figure this out, and avoid adding flags to `jsvu` as a workaround.

Reply:
This is such a low priority at this point - I'm unblocked to use jsvu where I need to - so I'm not sure we should worry about this for now. Maybe another time.